tools/mason/MasonHelp.chpl

```diff
@@ -403,3 +405,43 @@ proc masonSystemPcHelp() {
 }
 
 
+proc masonPublishHelp(){
+  writeln("Publish a package to the mason-registry repository");
+  writeln();
+  writeln("Usage:");
+  writeln("    mason publish [options]");
+  writeln();
+  writeln("Options:");
+  writeln("    -h, --help                  Display this message");
+  writeln(); 
+}
+
+
+proc masonCleanHelp(){
+  writeln("Uninstall mason from a mason package");
+  writeln();
+  writeln("Usage:");
+  writeln("    mason clean [options]");
+  writeln();
+  writeln("Options:");
+  writeln("    -h, --help                  Display this message");
+  writeln();
+  writeln("To remove mason completely, change directory to $CHPL_HOME/tools/mason before invoking mason clean");
```

Comment:
If we were to keep this string, I'd suggest breaking it up due to exceeding 80 chars. However, I think we need to change the help output here.